### ISSUE TITLE
Background card synchronization error

### ISSUE THREAD

**Issue by ao-apps** [aoindustries/ao-payments-stripe#1]
```
                com.stripe.exception.InvalidRequestException
                    Message...........: This property cannot be expanded (sources). You may want to try expanding 'data.sources' instead.; request-id: req_********

                        at com.stripe.net.ApiResource.requestCollection(ApiResource.java:186)
                        at com.stripe.model.Customer.list(Customer.java:220)
                        at com.aoindustries.creditcards.stripe.Stripe.getTokenizedCreditCards(Stripe.java:2288)
                        at com.aoindustries.creditcards.CreditCardProcessor.synchronizeStoredCards(CreditCardProcessor.java:527)
                        at com.aoindustries.aoserv.master.PaymentHandler.synchronizeStoredCards(PaymentHandler.java:1968)
                        at com.aoindustries.aoserv.master.PaymentHandler.access$3100(PaymentHandler.java:80)
                        at com.aoindustries.aoserv.master.PaymentHandler$1.run(PaymentHandler.java:2002)
                        at com.aoindustries.cron.CronDaemon$CronJobTask.run(CronDaemon.java:201)
                        at com.aoindustries.util.concurrent.ThreadLocalRunnable.run(ThreadLocalRunnable.java:51)
                        at com.aoindustries.util.concurrent.ThreadLocalsRunnable.run(ThreadLocalsRunnable.java:66)
                        at com.aoindustries.concurrent.Executors.lambda$incompleteFutureSubmit$1(Executors.java:533)
```

**Comment by ao-apps** [comment]
Fixed https://github.com/aoindustries/ao-payments-stripe/commit/e42d7debd65d53a0291aded7896c73356c441053

**Comment by ao-apps** [comment]
Fix included in release 2.0.0: https://github.com/aoindustries/ao-payments-stripe/releases/tag/ao-payments-stripe-2.0.0
